bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/Arduino/NibeGW/NibeGW.ino

```diff
@@ -227,6 +259,7 @@ void initializeEthernet()
 {
   Ethernet.begin(mac, ip, gw, mask);
   ethernetInitialized = true;
+  udp.begin(TARGET_PORT);  
```

Comment:
Target port is port where udp datagrams are sent and as arduino sw don't receive anything from that port, we shouldn't need to initialize that (begin function initialize port for listening at least in standard library).

Reply:
As mentioned in #9846 I had to add this line to get it working with arduino uno r3 and W5100-Eth-Shield. Otherwise no udp packets were sent. BUT I don't know if platformio is using other libs. On the weekend I will test it again using Arduino IDE.

Reply:
Did you a have time to test this?

Reply:
No, sorry. Please give me three more days. Thx :)
I postponed it because now I have W5500-Arduino-Shield AND W5100-Arduino-Shield to test several HW and SW combinations. Maybe they also act differently.

Reply:
With original Arduino IDE it seems to work. Should I remove this line and push it again?

Reply:
@paulianttila Can you give an advice here?

Reply:
I needed to compile my ProDiNo board again with latest IDE and ProDiNo libraries. I faced some weird issues as well and need to make some improvements and also add this line to make it reliable. I will make a PR about those changes, so I'm fine to add udp.begin here as well.